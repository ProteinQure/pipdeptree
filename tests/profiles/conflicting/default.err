Warning!!! Possibly conflicting dependencies found:
* Jinja2==3.0.3
 - MarkupSafe [required: >=2.0, installed: 0.22]
------------------------------------------------------------------------
