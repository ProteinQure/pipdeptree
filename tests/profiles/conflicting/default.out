argparse==1.4.0
Flask==0.10.1
  - itsdangerous [required: >=0.21, installed: 0.24]
  - Jinja2 [required: >=2.4, installed: 3.0.3]
    - MarkupSafe [required: >=2.0, installed: 0.22]
  - Werkzeug [required: >=0.7, installed: 0.11.2]
pipdeptree==2.2.0
  - pip [required: >=6.0.0, installed: 20.2.3]
setuptools==59.6.0
wheel==0.37.1
